src/domain/event/eventDetailsPage.module.scss

```diff
@@ -49,3 +49,6 @@
     color: var(--color-delete-icon);
   }
 }
+
+.futureOccurrencesList {
```

Comment:
Should there be some styling? Maybe you could remove this if not

Reply:
almost added some styles but didn't, forgot this one here, will remove